_tests/test_utils.py

```diff
@@ -27,6 +27,6 @@ def test_slugify(self):
 
     def test_expand_path(self):
         # noinspection SpellCheckingInspection
-        self.assertEqual(utils.expand_path("$HOME/whatever"), "/home/deutschegabanna/whatever")
+        self.assertFalse(utils.expand_path("$HOME/whatever").startswith("$HOME"))
```

Comment:
Can you please explain why is this a better comparison? In my case, I was comparing a function result with string literal. You are comparing a function result without variables with an unresolved variable.
The expected result will be:
`/home/runner/whatever != start with $HOME`

Reply:
ha! that is a trap! on windows it's `C:/Users/deutschegabanna` and on osx it will be `/Users/deutschegabanna/` so it's not that easy to test that ;)

so basically I give "$HOME" variable to my method and I expect it to be gone (expanded to something else, I don't really care what - since every system can have their own value. I just care, that it got replaced)

Reply:
right, but even if `$HOME` is **not** replaced, the following comparison will still be true.

Reply:
aah, ok, now I see. `assertFalse` so that it **has to be gone**. Thanks